code/modules/mob/living/carbon/superior_animal/attack.dm

```diff
@@ -164,7 +164,7 @@
 			var/offset_temp = right_before_firing()
 			A.launch(target, def_zone, firer_arg = src, angle_offset = offset_temp) //this is where we actually shoot the projectile
 			right_after_firing()
-			SEND_SIGNAL(src, COMSIG_SUPERIOR_FIRED_PROJECTILE, src, A)
+			SEND_SIGNAL(src, COMSIG_SUPERIOR_FIRED_PROJECTILE, A)
```

Comment:
These are real linter fails

Reply:
I disagree, I just think linters are outdated. Its not expecting the [ after the ?, despite ?[] being a valid operator introduced in 514.